### DISCUSSION TITLE
인간 미플 만들기 프로젝트 | Stop Bored Play Board

### DISCUSSION THREAD

**Discussion by giscus** [konsent/konsent.github.io#5] [Announcements]
# 인간 미플 만들기 프로젝트 | Stop Bored Play Board

보드게임 일기장

https://konsent.github.io/2023/08/23/baby/

**Comment by RominaSCH** [comment]
만 5세쯤 되니까 할만한게 쑥쑥 늘더라구요 ㅋㅋ 고생많았다 좀 더 크자... 빨리 한글 배워와 게임하게!!!
